articles/quickstart/webapp/php/01-login.md

```diff
@@ -11,159 +12,214 @@ topics:
 contentType: tutorial
 useCase: quickstart
 github:
-  path: 00-Starter-Seed
+  path: .
```

Comment:
same comment about using folders

Reply:
See previous explanation 👍

Reply:
@evansims I believe this change is what caused the issue reported in slack today.